release_bot/utils.py

```diff
@@ -258,3 +259,17 @@ def update_version(file, new_version, prefix):
             output.write('\n'.join(content) + '\n')
         configuration.logger.info('Version replaced.')
     return changed
+
+def pypi_project_from_setup_cfg():
```

Comment:
I don't think there's any use for this, now.

Reply:
I think there is. This PR should fix #170, which we hit because [packit](https://github.com/packit-service/packit)'s PyPI name is [packitos](https://pypi.org/project/packitos/).
Once it's merged, release-bot will take the PyPI name from [setup.cfg](https://github.com/packit-service/packit/blob/master/setup.cfg#L2). If we remove this `pypi_project_from_setup_cfg()` we would need to add the `pypi_project: packitos` into config file.

Reply:
I meant it's duplicate code. It is also present in [configuration.py](https://github.com/Elias999/release-bot/blob/170issue/release_bot/configuration.py#L56).

We can get rid of either.

Reply:
My intention was to import it from utils but for some reason i got an error when i try this.
So i let it so i can discuss it with someone

Reply:
I'd say it was correct to have the function in utils, importing it like this should work just fine: `from release_bot.utils import pypi_project_from_setup_cfg`

Reply:
Here is the problem that throws when i import it from utils
![error-release](https://user-images.githubusercontent.com/36102669/55689900-22c65a80-5993-11e9-8a62-03981ced8014.png)

Reply:
that's an error caused by circular dependency: you import configuration from utils while configuration imports from utils: utils should not depend on anything in the project: the solution should ideally be that the function in utils would not need configuration and would get the values it needs directly